playwright/tests/fixtures.ts

```diff
@@ -0,0 +1,13 @@
+import { playwrightTest } from "@axe-core/watcher";
+
+const API_KEY = process.env.AXE_API_KEY || "";
+
+const { test, expect } = playwrightTest({
```

Comment:
Do you mind explaining a bit what this wrapper does and how we'd expect to use it in tests. Either documented in here and/or in a testing section of the README could be useful, it's not super clear from just the code what this is testing or what we expect to see from this tool.

EDIT: oops just realized this is still in draft mode -- apologies if that is in the works and I gave this feedback too early.